kms/attestations/verify_attestation_test.py

```diff
@@ -0,0 +1,102 @@
+MSMwIQYDVQQLDBpjbG91ZC1rbXMtcHJvZC11cy1jZW50cmFsMTEXMBUGA1UEAwwO
+VGVzdCBQYXJ0aXRpb24wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDh
+cOk8hil8G4vl66jb02eXu2rKO6PmNF1GziXtzawyhx/+PDmEuGu+g/8hpivY6vDr
+buAMFs8OIBBBlfED/soVANflXktDRGqWJau+rIrw9EGcclcwzlIboIi6KLPcLih0
+N0TTxqRLgy2EsEQ6UKS7su4bOIxD3Z6FSeTsoq+C2sgSWXmLitO0iRYYcgRjoyCU
+kdzzO/JCwPKhhQx5NUrrHseALaIltG4D0aWLuBZKyV38yA1XEMdyCGk7RedEYC+v
+OzaJrNToQBCIaCdn3F0uqJd49irLNPyQ5CY3NNL8rupJSq3iVxhEIZ8ANaU8UDvo
+5iaQNKV1/KiQsXfUW6fbAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAIIGB0aXhd6k
+xyWgYzJ0DHpYP2ALwHRWXde5PtEkdHDguPlUTzZ5iTfqOJlEeUXlDO9RV82jc4mE
+hguCaMl3Q+2JGLJLCnSDgcaY5WAVBF9FSakdbHBj4Ik9L8NDlUQB6Me4d4gKWpg1
+bUD4n2KtvCZGZzA3pfRBcYyAbwC+xEi1XrITyshb0pkjsWO4Urg36W/RpkCiYAw0
+Xua0jJMG/wcF+xktd7kgcsBh5Es2VCzyQwisXoOIi3EY7lMJK2+ctjQFy1GxumBU
+jBlXj0VjAm3QOVLTh3mfb1XofoIjOOYkMBjXMiQhFy/Lv68u5q7qlEYe92OKxkCO
+0UaAcqt8+QM=
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIDRjCCAi4CFBVm+eV+oRkaYq2NyuTfwxWapjFOMA0GCSqGSIb3DQEBCwUAMGYx
+CzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRl
+cm5ldCBXaWRnaXRzIFB0eSBMdGQxHzAdBgNVBAMMFlRlc3QgTWFudWZhY3R1cmVy
+IFJvb3QwHhcNMjAwMzMxMjE0MjU1WhcNMzAwMzI5MjE0MjU1WjBZMQswCQYDVQQG
+EwJBVTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lk
+Z2l0cyBQdHkgTHRkMRIwEAYDVQQDDAlUZXN0IENhcmQwggEiMA0GCSqGSIb3DQEB
+AQUAA4IBDwAwggEKAoIBAQC28Wu0dusN6AYkYon8RIuHlJWWwZWlTxXSMK4v/IOY
+pG9F2/gUEDMQOgpyCCpTc5eLHRPa/Z2QgB0c2VSlQC8FZ1l9/YL7uBTJ0UpDoBf8
+LUimIqotneXpL+7CW1kWFLZIgpm0iVuTPjV2b3frtvu0B+nYuyo4dtToebqoOKse
+F3ymLsAjSqA9aoCD0XbspAkLJIvdQU28vXY4Y2y0OTGUnaQ7ZDwkNLxeAfeIdNJD
+FRCcYsLRopsyptFMYLLDrI70gywAGYaGOxYG8747BIZELyT5Gnt0o7JwpuF8Mi53
+T5NGiu5/wLwXnxRRhb3M5+lStdTfvbEfgK1mC0ac8ym5AgMBAAEwDQYJKoZIhvcN
+AQELBQADggEBAILH0Q8WlgaGBosPBjnpAsp4eZOfq9skKZERYzFA2wBAy4B0q9/S
+3oL1MIZEU6/ckiFyAm3r4/ZxMLX0UrisuRQUFQ3w+gqFccqlmGETsZGJpPtfIm+n
+JQo44XXnTHndqoYPNfvfQkp0WtJQS8hSgTASP+1GYjqOn1fZaa28m39/mx9Mrw7K
+xtXOtrdKqJhWCWJPprfC5sYNCYTA2HXVmBU2Y4AP4A4w+A1gCAdzvH8EEyDjnvxJ
+GEa4uczhA3n+NmhLipg1aGbxJO5ZHXdyFF2rTXVVXSiX8EEasnwcTDjeXDKhdpu6
+biaxW9fnsJIXirAE03FFkC/tWelkGFkmMfs=
+-----END CERTIFICATE-----"""
+
+TEST_ATTESTATION_B64 = """
+Fad+V9p6SHEwM5XRRs8dCuC7dhHtrhg2wMwuzynxP/ch89ZNhf65hLIIVjIooYddqwE9tQ8pfgbu
++i+UpniWb7HLJIKQ4DNKCQPwpKWp+bLO3TL6bZRXBwB+pcLNcaGBGIPg2RFrXb34gUCcKoCRUrCu
+nYi4VNE+9jvkg3ElX4p3iTS1Oupi0pqB3ab5lP84se1/cw7A3okAXY9MgooRj1zkODOdJgsl/Q50
+j6gatUu0x5bRfQbdk2kfwUCS730oQg/RA6pZb5utqXp3I8iarZT8B3FdeOuiQ0H4rJbZ5Xk0rl2B
+sCSTCXjbzAg6JR3icarIOsI8Q7Wb0bA8EoqZZhGaIlEdXQ=="""
+
+TEST_ATTESTATION_B64_GZ = """
+H4sICNrehF4AA2F0dGVzdGF0aW9uAAEGAfn+Fad+V9p6SHEwM5XRRs8dCuC7dhHtrhg2wMwuzynx
+P/ch89ZNhf65hLIIVjIooYddqwE9tQ8pfgbu+i+UpniWb7HLJIKQ4DNKCQPwpKWp+bLO3TL6bZRX
+BwB+pcLNcaGBGIPg2RFrXb34gUCcKoCRUrCunYi4VNE+9jvkg3ElX4p3iTS1Oupi0pqB3ab5lP84
+se1/cw7A3okAXY9MgooRj1zkODOdJgsl/Q50j6gatUu0x5bRfQbdk2kfwUCS730oQg/RA6pZb5ut
+qXp3I8iarZT8B3FdeOuiQ0H4rJbZ5Xk0rl2BsCSTCXjbzAg6JR3icarIOsI8Q7Wb0bA8EoqZZhGa
+IlEdXayB4gUGAQAA"""
+
```

Comment:
Can you add brief comments on how to generate these values so that others can add similar values in the future?